app/services/hyrax/thumbnail_path_service_decorator.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# OVERRIDE Hyrax v5.0.0rc2 - use site defaults instead of app wide defaults
+module Hyrax
+  module ThumbnailPathServiceDecorator
+    def default_image
+      Site.instance.default_work_image&.url || super
```

Comment:
Yummy refactor!

Reply:
It looks like this is a complete duplication of the `WorkThumbnailPathServiceDecorator`